protos/flyteidl/core/tasks.proto

```diff
@@ -154,4 +154,44 @@ message Container {
     // Ports to open in the container. This feature is not supported by all execution engines. (e.g. supported on K8s but
     // not supported on AWS Batch)
     repeated ContainerPort ports = 7;
+
+    // BETA: This enables use of CoPilot. This makes it possible to to run a completely portable container, that uses inputs and outputs
```

Comment:
nit: generally speaking, I do not like using code names in code.... flyte (the product name/identity) is the exception... I would replace CoPilot (also casing looks weird :D) with what it actually does...

Reply:
data?